### PR TITLE
Add pa11y

### PR DESCRIPTION
**Pull request by ricardoom**
thanks team for helping met get this sorted out. while some of the code fails, the tests work! onward.

### CODE REVIEW

**Comment by ricardoom** [comment]
@rhernand3z you can consider the review optional...

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# matrix:
```
Comment (EdwinGuzman) [inline .travis.yml]: You can delete these commented out lines.

File: INSTALL.md [hunk 2]
```diff
@@ -100,4 +104,7 @@ Avoid using the `*= require_self`, `*= require_tree .`, and `*= require` stateme
   @import "nypl-normalize";
   ```
 
-**Important:** Make sure the file has `.scss` extension (or `.sass` for Sass syntax).
\ No newline at end of file
+**Important:** Make sure the file has `.scss` extension (or `.sass` for Sass syntax).
+
+## <a name="pa11y"></a>Accessiblity Testing with pa11y
```
Comment (EdwinGuzman) [inline INSTALL.md]: You should add that you need to start a server at localhost:4000 before running the tests.

Comment (EdwinGuzman) [reply INSTALL.md]: And that you need to run `npm run test-pa11y`.

Comment (ricardoom) [reply INSTALL.md]: I'll add instructions to the install docs...

**Comment by EdwinGuzman** [comment]
It seems to be working well!

**Pull request merged**